client/src/util/locationConverter.ts

```diff
@@ -0,0 +1,37 @@
+import { COORDINATE_PRECISION } from '../components/SearchBar/constants.ts';
+import { Location } from '../gql/graphql.ts';
+
+const DOUBLE_PATTERN = '-{0,1}\\d+(\\.\\d+){0,1}';
+
+const LAT_LON_PATTERN = '(' + DOUBLE_PATTERN + ')(\\s*,\\s*|\\s+)(' + DOUBLE_PATTERN + ')';
```

Comment:
The toString method outputs a different format than what is expected from the parser. Do we really want that?

Reply:
I wanted to both preserve the original output format and be consistent with the java `LocationStringParser`, which allows both `lat lon` and `lat, lon` formats. The latter one will be converted to the sooner one by the `toString` and it can be confusing. 

Do we want to support both formats, or should I remove the latter one?

Reply:
Besides, the Transmodel API which the debug UI uses doesn't use LocationStringParser.

Reply:
I removed the `lat, lon` format.